src/scss/_header.scss

```
@@ -40,6 +40,39 @@
             }
         }
     }
+
+    &.registry {
+        $bgColor: #2b2b2c;
+
+        background-color: $bgColor;
+
+        .logo-nav-menu {
+            @apply w-full;
+
+            background-color: $bgColor;
+
+            @screen xs {
+                @apply pt-2 -ml-2;
+            }
+
+            @screen md {
+                @apply w-auto absolute z-50 rounded rounded-t-none pb-2;
+                top: 4rem;
+            }
+
+            li {
+                @apply text-lg text-white p-2;
+
+                &:hover {
+                    @apply underline;
+                }
+
+                a {
+                    @apply block w-full;
+                }
```

Little-ish thing I just noticed: putting the padding and the hover handler on the `li `, rather than on the the `a`, is making for a slightly odd interaction in which you get underlines without click targets. It might be better if the list items had no padding at all and the anchors did, such that if something were underlined, it'd always be clickable. 

```suggestion
                @apply text-lg text-white;

                a {
                    @apply block px-4 py-2;

                    &:hover {
                        @apply underline;
                    }
                }
```

☝️ Also adds a bit more horizontal padding to the list items, as they feel a bit squeezed compared to the design.